### PR TITLE
Add hover effects + misc fixes

### PR DESCRIPTION
**Pull request by OceanOak**
This PR:
- Adds shine effect to labels, 
- Makes the toggle headings clickable.
- Fixes code overflow on mobile view 

### Shine effect:

https://github.com/darklang/darklang.com/assets/87861924/943b9fdc-b965-455c-8210-3eaee5b39795

### Clickable toggle headings:

https://github.com/darklang/darklang.com/assets/87861924/530eef9f-a69e-4ccd-9552-bab502d758fb

### Mobile View:
Before:
Screenshot 2023-11-13 at 15 54 44

After:
Screenshot 2023-11-13 at 15 55 40

### Suggestion (Not committed)
I think it looks better, but I could be wrong. Feedback is appreciated

Increased font size:

https://github.com/darklang/darklang.com/assets/87861924/e49b0bda-02ab-4d3e-927f-290f3e2e322d

Navbar blur effect: (Not included in this PR)

https://github.com/darklang/darklang.com/assets/87861924/c35ac0c7-bde9-4853-830e-6bbe532b3167

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -73,72 +73,71 @@
...
-                    .forEach(element => {
-                      count = count + 1;
-                      element.setAttribute(attr.name, attr.value);
-                      element.removeAttribute(placeholderAttribute);
-                    });
-                  if (count > 0) {
-                    this.removeAttribute(attr.name);
-                  }
-                });
+    <script defer type="module">
```
Comment (OceanOak) [inline public/index.html:77]: Replaced `document.addEventListener("DOMContentLoaded", () => ` with `defer`.  It accomplishes the same functionality.

Comment (pbiggar) [reply public/index.html:77]: Great, thanks!

**Comment by OceanOak** [comment]
> Everything committed here looks great! Regarding the suggested blurred top header, I'm torn in opinion.

I'll remove it for now, to get this merged. It's just three Tailwind classes, so it'll be easy to add back if needed.
We will probably make more changes to the homepage soon

**Review by pbiggar — APPROVED** [review]

File: public/index.html [hunk 2]
```diff
@@ -661,7 +661,7 @@
 >   Executes any file
 > Continue? [N, y] y
 > These are unusual permissions. Are you sure [N, y] y
-Ha ha, you're own3d
+Ha ha, you're owned
```
[the remark this answers is not in the document]
Comment (OceanOak) [reply public/index.html:662]: Oh, sorry. I'll fix it

**Pull request merged**